Traceback (most recent call last):
  File "/Users/markcrovella/opt/anaconda3/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/Users/markcrovella/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/markcrovella/opt/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/markcrovella/opt/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/markcrovella/opt/anaconda3/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/Users/markcrovella/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/Users/markcrovella/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/markcrovella/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
string = "string"
string[-1] = "y"  # This will generate and error as we attempt to modify the string
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-26-b5fa142bf995>[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0mstring[0m [0;34m=[0m [0;34m"string"[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mstring[0m[0;34m[[0m[0;34m-[0m[0;36m1[0m[0;34m][0m [0;34m=[0m [0;34m"y"[0m  [0;31m# This will generate and error as we attempt to modify the string[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;31mTypeError[0m: 'str' object does not support item assignment
TypeError: 'str' object does not support item assignment

